### ISSUE TITLE
[Craft 4]: Only first supertable is rendered

### ISSUE THREAD

**Issue by magicspon** [engram-design/SuperTable#471]
**Description**
If you have more than one supertable on a single entry type, only the first table will appear.

**Steps to reproduce**
1. Create two supertables
2. Add both to an entry type
3. Observe

![image](https://user-images.githubusercontent.com/3268717/171404605-2f89c1de-aec7-4635-bd1a-a4cee3fceaf5.png)
![image](https://user-images.githubusercontent.com/3268717/171404698-7f8b7dba-281d-4d8a-b2f2-57ab00299fda.png)

**Additional info**
- Plugin version: 3.0.0-beta.4
- Craft version: 4.0.3
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Can't replicate this sorry, I've got dozens of entry types setup like this. Can you let me know your Super Table settings (layout, and if static)? Also worth trying on the latest `craft-4` branch as there was a similar issue.

**Comment by magicspon** [comment]
it only happens when they are both static.

**Comment by engram-design** [comment]
Still can't replicate this sorry, I've tried all the different layouts, more than 2 fields, new entries, drafts, provisional draft, existing entries.

There was a similar fix https://github.com/verbb/super-table/issues/464 if you can try that also. Change your `verbb/super-table` requirement in `composer.json` to:

```
"require": {
  "verbb/super-table": "dev-craft-4 as 3.0.0-beta.4",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
This should be fixed in [3.0.0-beta.5](https://github.com/verbb/super-table/releases/tag/3.0.0-beta.5)

**Comment by magicspon** [comment]
aces... thanks @engram-design, I've not been able to check, turbo busy!
